tests/sentry/web/frontend/test_2fa.py

```diff
@@ -3,13 +3,14 @@
 from django.core.urlresolvers import reverse
 
 from sentry.testutils import TestCase
-from sentry.models import TotpInterface
+from sentry.models import TotpInterface, User
 
 
-class TwoFactorAuthTest(TestCase):
+class TwoFactorAuthTestPassword(TestCase):
 
     def test_security_renders_without_2fa(self):
-        user = self.create_user('foo@example.com')
+        User.objects.create(email='a@example.com')
```

Comment:
Yeah create_user is just a helper function so you dont have to import User, and it also sets sane defaults (though for user thats less relevant).

Functionally they should be the same barring the email change.